services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from typing import Any
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: Any,
```

Comment:
Consider using a more specific type hint for `movie_show_time` instead of `Any`. Since `show_time` is a `DateTimeField`, using `datetime` as the type hint would be more appropriate.